var/spack/repos/builtin/packages/mfem/package.py

```diff
@@ -448,6 +474,18 @@ class Mfem(Package, CudaPackage, ROCmPackage):
             "amgx~mpi cuda_arch={0}".format(sm_), when="+amgx~mpi cuda_arch={0}".format(sm_)
         )
 
+    # double precision requirements, may be incomplete:
```

Comment:
(question) in what sense do you mean incomplete? `["@:4.6", "precision=double"]` appears to partition the space of possibilities based on what you added. Do you mean there might be some versions <= 4.6.... that would want to use `hypre precision=single`?

Reply:
The "incomplete" refers to the list of dependencies to which we need to propagate the "precision=double" setting -- I did not dig deeper to find all optional packages that can be built with precision other than `double`.

Reply:
Thanks, could you add that detail to the comment? I assume the idea is that `mfem` might benefit from additional dependencies building with different precisions, but for now it doesn't seem to need this. If I misunderstood, and this isn't for `mfem`'s benefit, then IMO it would be clearer to remove the comment.

Reply:
I moved and extended the comments (for both `double` and `single` precision) just before the respective `depends_on(...)` directives in b5a729faec. Does this make things clearer?

Reply:
Ok, your suggestion sounds good to me too -- I updated the comment in 0c61c4ded4.